Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -1827,12 +1826,36 @@ public void setIdentity(@NonNull String userId, @Nullable BranchReferralInitList
         }
     }
 
+    /**
+     * Gets all available cross platform ids.
+     *
+     * @param callback An instance of {@link io.branch.referral.util.BranchLastAttributedTouchData} to callback with cross platform ids
+     *
+     */
```

Comment:
Probably too late to comment on this, however I don't like how the constructor is making a network request as a side effect.

Reply:
Fixed.